deployment/celery_config.py

```diff
@@ -21,9 +21,11 @@
 worker_name_default = 'worker_default@ww_host'  # check /etc/default/celeryd
 worker_name_user = 'worker_user@ww_host'
 # task_soft_time_limit = 4  # seconds
-default_task_soft_time_limit = 3600  # 60 minutes
+# default_task_soft_time_limit = 3600  # 60 minutes
+default_task_soft_time_limit = 120  # 2 minutes, debugging, TNT
```

Comment:
If we're going to keep it this way, I'd suggest removing the `debugging, TNT` bit here and on line 28. If it is still only for debugging, then I don't think there's a need to commit these changes right now.

Reply:
oops, yup! I've removed those comments — I think we could be bold here and commit these changes (at the very least, nothing has got _worse_, and I think it has potentially got _better_?)